src/ansys/fluent/core/session.py

```diff
@@ -76,6 +78,11 @@ def __call__(self):
         return self._scheme_eval.scheme_eval("(data-valid?)")
 
 
+def _get_uploader(fluent_connection: FluentConnection):
+    uploader = PimFileTransferService(fluent_connection._remote_instance)
+    return uploader
```

Comment:
If we are keeping `_get_uploader`, I think this is a little cleaner:
```suggestion
    return PimFileTransferService(fluent_connection._remote_instance)
```

Reply:
Also, couldn't this `uploader` be just an attribute of the `session` or `fluent_connection` objects? Don't we already have something similar defined or am I misremembering?

Reply:
We don't want to introduce new dependency of `PimFileTransferService` in `session` or `fluent_connection`, it is a separate object.